lib/syscall_shim/include/uk/syscall.h

```diff
@@ -380,20 +432,30 @@ typedef long uk_syscall_arg_t;
 	static inline rtype __##rname(UK_EXECENV_DECLMAPx(UK_S_EXECENV_ARG_ACTUAL,\
 						      x, UK_S_ARG_ACTUAL,\
 						      __VA_ARGS__));	\
-	long __used rname(long _execenv)					\
+	long __used rname(long _execenv)				\
 	{								\
 		struct ukarch_execenv *execenv;				\
 		long ret;						\
 									\
 		execenv = (struct ukarch_execenv *)_execenv;		\
+									\
```

Comment:
There's something buggy in the R_E variant, as the handlers are executed twice: Once upon entry to `ukplat_syscall_handler()` and once again upon execution of the actual syscall.

Reply:
Yes they should be moved to the ename variants.